packages/livechat/src/routes/Chat/container.js

```diff
@@ -288,8 +288,8 @@ class ChatContainer extends Component {
 	};
 
 	canFinishChat = () => {
```

Comment:
What do you think of naming this `canCloseChat` for the sake of consistency?

Reply:
We use this term close/finish indiscriminately on the codebase :/ i can for sure rename "visitorCanCloseChat" to "visitorCanFinishChat"

(I didnt modify this func tho, was already there)